src/NzbDrone.Core.Test/ParserTests/ParsingServiceTests/AugmentersTests/AugmentWithOriginalLanguageFixture.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Core.Languages;
+using NzbDrone.Core.Parser.Augmenters;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.Test.ParserTests.ParsingServiceTests.AugmentersTests
+{
+    [TestFixture]
+    public class AugmentWithOriginalLanguageFixture : AugmentMovieInfoFixture<AugmentWithOriginalLanguage>
+    {
+        [Test]
+        public void should_add_movie_original_language()
+        {
+            var releaseInfo = new ParsedMovieInfo();
+            var remoteMovie = new RemoteMovie
+            {
+                Movie = new Movies.Movie
+                {
+                    OriginalLanguage = Language.English
+                }
+            };
+            var result = Subject.AugmentMovieInfo(releaseInfo, remoteMovie);
+            result.ExtraInfo.Should().ContainKey("OriginalLanguage");
+            result.ExtraInfo["OriginalLanguage"].Should().Be(Language.English);
+        }
+    }
+}
```

Comment:
Add a new line at the end of the file

Reply:
It might be hidden by the GitHub diff : the new line is here on my commit, and also if I edit the file on Github.